Formula/v8@5.8.rb

```diff
@@ -0,0 +1,118 @@
+# Track Chrome stable.
+# https://omahaproxy.appspot.com/
+class V8AT58 < Formula
+  desc "Google's JavaScript engine"
+  homepage "https://github.com/v8/v8/wiki"
+  url "https://chromium.googlesource.com/chromium/tools/depot_tools.git"
```

Comment:
Ideally, it should be, what's here is dirty hack.

The issue that cloning v8 repo with git doesn't make sense as the only supported way to do that is to use `gclient`, at least that's what v8 docs says (https://github.com/v8/v8/wiki/Using%20Git).

So here we may cache gclient synced v8 folder to reuse it later.